piety-promoted-v3/modernized/05-chapter-05.adoc

```diff
@@ -668,11 +668,14 @@ being nearly fifty-four years of age.
 [.asterism]
 '''
 ISAAC SHARPLES.
---A short narrative of ISAAC SHARPLES, late of Hitchin,
+--A short narrative of Isaac Sharples, late of Hitchin,
```

Comment:
Not sure if this should be a heading?

Reply:
yeah, it's a tough one too.  I think let's move it inside of the embed block and make it a sub-heading.  so there will just be the line 670 above with just the name, then the embed.